packages/eslint-plugin/tests/rules/prefer-literal-enum-member.test.ts

```diff
@@ -0,0 +1,207 @@
+import rule from '../../src/rules/prefer-literal-enum-member';
+import { RuleTester } from '../RuleTester';
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+});
+
+ruleTester.run('prefer-literal-enum-member', rule, {
+  valid: [
+    `
+enum ValidRegex {
+  A = /test/,
+}
+    `,
+    `
+enum ValidString {
+  A = 'test',
+}
+    `,
+    `
+enum ValidNumber {
+  A = 42,
+}
+    `,
+    `
+enum ValidNull {
+  A = null,
+}
+    `,
+    `
+enum ValidPlain {
+  A,
+}
+    `,
+    `
+enum ValidQuotedKey {
+  'a',
+}
+    `,
+    `
+enum ValidQuotedKeyWithAssignment {
+  'a' = 1,
+}
+    `,
+    // eslint-disable-next-line @typescript-eslint/internal/plugin-test-formatting
+    `
```

Comment:
`noFormat` is a template tag that's exported from `../RuleTester`, and it will disable formatting for this string

```suggestion
    noFormat`
```

Reply:
Didn't know about that.  Will do.